.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: UnitTests
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: true
+            matrix:
+                include:
+                    -   php: '5.5'
+                        dependency: lowest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.5'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.6'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.0'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.1'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: '^3.0'
+                    -   php: '7.2'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^4.0'
+                    -   php: '7.3'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
+                    -   php: '7.4'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
+
+        steps:
+            -   name: Checkout source
+                uses: actions/checkout@v2
+
+            -   name: PHP setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: ${{ matrix.coverage }}
+
+            -   name: Require Symfony version
+                if: matrix.symfony != 'any'
```

Comment:
```suggestion
                if: matrix.symfony
```

and then don't define it with `any` value

Reply:
I am not sure how will work naming in this case.  But I can change `any` to `*` to be familiar with composer syntax 
![image](https://user-images.githubusercontent.com/4967813/141688628-9ed9ec43-7b62-4ce5-a0e5-0b2b47f4a787.png)

Reply:
I want to leave this `any` value to provide pretty naming for the jobs. If you still want to remove it - let me know

Reply:
:+1:

then, maybe replace with `*` to mimic composer versionioning and then we have `Symfony@^4` or `Symfony@*`